test/test.js

```diff
@@ -3301,6 +3303,30 @@ describe('Page', function() {
       }
     });
   });
+
+  describe('Response', function() {
+    it('specifies when response comes from cache and service worker', async() => {
```

Comment:
page and browser are passed into the test:

```js
async ({page, browser}) => {
  // ..
});
```

No need to create another one; reusing browser speeds up tests significantly

Reply:
I was doing that but it was causing other tests to fail: https://travis-ci.org/GoogleChrome/puppeteer/jobs/317396566#L556